src/Compilers/CSharp/Portable/Syntax/SyntaxKindFacts.cs

```diff
@@ -1592,6 +1593,8 @@ public static string GetText(SyntaxKind kind)
                     return "when";
                 case SyntaxKind.InterpolatedVerbatimStringStartToken:
```

Comment:
@CyrusNajmabadi `GetText` is a public API. If I tried to have a single syntax kind for both `$@` and `@$`, it would be a break (although a small one).